config/config.go

```diff
@@ -321,6 +330,54 @@ func validate() error {
 	return nil
 }
 
+func GetHostname() (string, error) {
+	return getHostname(
+		vip.GetString(OperatorExtraIPKey),
+		vip.GetString(OperatorExtraDomainKey),
```

Comment:
These vars are actually string slices, not string
```suggestion
		vip.GetStringSlice(OperatorExtraIPKey),
		vip.GetStringSlice(OperatorExtraDomainKey),
```

Reply:
Done